docs/wiki/dev-guides/Roadmap.md

```diff
@@ -0,0 +1,9 @@
+We do not have a formal roadmap, but we're happy to discuss features, improvements, new adapters, etc, in our [discussions area](https://github.com/Point72/csp/discussions). Here are some high level items we hope to accomplish in the next few months:
```

Comment:
@ptomecek looks like we still need to update this

Reply:
@robambalu Yeah, why don't you put your thoughts in a PR and I'll comment anything that I think might be missing? I think last time we discussed, @trallard suggested taking a longer-term view (was it a one-year horizon, or am I mis-remembering?)

Reply:
https://github.com/Point72/csp/discussions/71 is also a good place to gather thoughts.

> `@trallard` suggested taking a longer-term view (was it a one-year horizon, or am I mis-remembering?)

Yes, that's correct. We can consider focus areas (e.g., new adapters, user experience, outreach, exploration, etc.) for the next 12-18 months, and list some specific tasks for each theme.

Reply:
@ptomecek added three items to the other PR